sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 4.0.0-preview.7 (2019-10-07)
+- Improved the general API based on feedback.
```

Comment:
We should say more here. What types of things were changed, where should people look to see the new API differences?